models/map.js

```diff
@@ -35,6 +35,12 @@ function MapModel () {
 
     emitter.on(events.ZOOM_TO, function (id) {
       const match = state.features.filter(f => id === f.properties._id)
+      console.log(match)
+      //If there are no-coordinates to zoom to, does nothing
+      if(!match[0] || (typeof match[0].geometry.coordinates === 'boolean' || match[0].geometry.coordinates[0]===null || match[0].geometry.coordinates[1]===null))
+      {
+        return
+      }
```

Comment:
similar suggestion to https://github.com/digidem/mapeo-webmaps-public/pull/15/files#r778395069 i.e. checking that the `coordinates` field is an array of 2 non-null values

separately, can avoid emitting the event that triggers this in the first place by doing this check in the relevant view (`/views/list.js`). the click handler that emits the event is called [here](https://github.com/digidem/mapeo-webmaps-public/blob/87736258dcfe3a31bca6c1fb5614f062f5f3c019/views/list.js#L77-L80) so it would be a matter of moving this logic there and conditionally calling the handler. think keeping it here in the model is fine though